lib/functions-api.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+function current_event_call($data) {
+  $current_event_id = IGV_get_option('_igv_site_options', '_igv_radio_evento_current');
+
+  // Check if no event is selected
+  if (empty($current_event_id)) {
+    return false;
+  }
+
+  // Get selected event data
+  $current_event = get_post($current_event_id);
+
+  // Get event's feat thumbnail
+  $event_thumbnail = get_the_post_thumbnail_url($current_event_id);
```

Comment:
I think you should at the very least put a basic size in here and add that size to the thumbs declaration. Otherwise easy to forget and causes real performance issues.